contracts/SafeProtocolRegistry.sol

```diff
@@ -5,66 +5,66 @@ import {Ownable2Step} from "@openzeppelin/contracts/access/Ownable2Step.sol";
 import {IERC165} from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
 
 contract SafeProtocolRegistry is ISafeProtocolRegistry, Ownable2Step {
-    mapping(address => ComponentInfo) public listedComponents;
+    mapping(address => IntegrationInfo) public listedIntegrations;
 
-    struct ComponentInfo {
+    struct IntegrationInfo {
```

Comment:
Different topic, but should an integration have a type? Or are this only PlugIns in the first place? Then we should rename this mapping.

Reply:
This is only for plugin.

Reply:
done in #29